### ISSUE TITLE
Set date ?

### ISSUE THREAD

**Issue by Klian** [felicegattuso/datedropper#16]
Hi,

How can I set a date by default ?

**Comment by coRz** [comment]
added a solution for that on my fork

**Comment by Golmote** [comment]
It seems @felicegattuso is working on it. https://github.com/coRz/datedropper/commit/b3fd2b24bc7fb620f95154d87bffc1b129f2896b

**Comment by felicegattuso** [comment]
Yeah guys, I really hope to publish the new 1.2 version between today and tomorrow. May the force be with you ;)

**Comment by vithanidarpan** [comment]
@felicegattuso 
Hi, I know you have updated code to set date in datedropper. But how to set the value in datedropper?
I mean how can I set my value in datedropper options? I have searched everywhere but not found any solution. 
Also you have not mentioned in the document. So please reply me on this.

**Comment by felicegattuso** [comment]
I'm currently working on version 2.0, which also solves your request, as well as having a multitude of new features! Stay

**Comment by tiagoporto** [comment]
@felicegattuso Its possible set date now??
